### PR TITLE
LPS-161369 Track reindex actions in the console without additional configuration/LPS-163490

### PR DESCRIPTION
**Pull request by wcao20170619**
https://issues.liferay.com/browse/LPS-161369

### CODE REVIEW

**Comment by wcao20170619** [comment]
ci:test:relevant

File: modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/background/task/ReindexSingleIndexerBackgroundTaskExecutor.java [hunk 1]
```diff
@@ -120,6 +120,10 @@ protected void reindex(String className, long[] companyIds)
 				ReindexBackgroundTaskConstants.SINGLE_START, companyId,
 				companyIds);
 
+			if (_log.isInfoEnabled()) {
+				_log.info("Reindexing started [companyId=" + companyId + "]");
```
Comment (BryanEngler) [inline modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/background/task/ReindexSingleIndexerBackgroundTaskExecutor.java:124]: @wcao20170619 lets add className to the messages as well, thanks!

**Comment by lipusz** [comment]
Also, please consider consistency too: https://issues.liferay.com/browse/LPS-163666?focusedCommentId=2763374&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-2763374

File: modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/background/task/ReindexSingleIndexerBackgroundTaskExecutor.java (hunk 1 above)
Comment (wcao20170619) [reply modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/background/task/ReindexSingleIndexerBackgroundTaskExecutor.java:124]: @BryanEngler , I am not sure what you want(example?) currently, we have the following.
[ReindexPortalBackgroundTaskExecutor:57] Reindexing started [companyId=20097]
2022-09-19 22:09:53.530 INFO [liferay/background_task-2][ReindexPortalBackgroundTaskExecutor:76] Reindexing ended [companyId=20097]

Comment (lipusz) [reply modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/background/task/ReindexSingleIndexerBackgroundTaskExecutor.java:124]: @wcao20170619 My understanding is that we want the name of the entity class being reindexed to be also logged, say `com.liferay.journal.model.JournalArticle`

Comment (BryanEngler) [reply modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/background/task/ReindexSingleIndexerBackgroundTaskExecutor.java:124]: i think for now only add the class name in the 2 logging statements in ReindexSingleIndexerBackgroundTaskExecutor.java

Comment (BryanEngler) [reply modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/background/task/ReindexSingleIndexerBackgroundTaskExecutor.java:124]: example: [companyId=20097,className=com.liferay.journal.model.JournalArticle]

Comment (wcao20170619) [reply modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/background/task/ReindexSingleIndexerBackgroundTaskExecutor.java:124]: OK, ONLY in ReindexSingleIndexerBackgroundTaskExecutor.java.

Comment (wcao20170619) [reply modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/background/task/ReindexSingleIndexerBackgroundTaskExecutor.java:124]: Do we need to change the following too for the spellchecker?
Start indexing dictionary for com/liferay/portal/search/dependencies/spellchecker/en_US.txt [companyId=0]

**Comment by wcao20170619** [comment]
> Also, please consider consistency too: https://issues.liferay.com/browse/LPS-163666?focusedCommentId=2763374&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-2763374

https://issues.liferay.com/browse/LPS-163666?focusedCommentId=2763750&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-2763750

**Comment by wcao20170619** [comment]
ci:test:sf

**Comment by joshuacords** [comment]
ci:forward

**Pull request closed without merging**